serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,167 @@
+      name: IMAGE
+      type: string
+  tasks:
+    - name: fetch-repository
+      params:
+        - name: url
+          value: $(params.git-url)
+        - name: subdirectory
+          value: ''
+        - name: deleteExisting
+          value: 'true'
+        - name: revision
+          value: $(params.git-revision)
+      taskRef:
+        kind: ClusterTask
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+    - name: build-image
+      params:
+        - name: IMAGE
+          value: $(params.IMAGE)
+        - name: TLSVERIFY
+          value: 'false'
+      runAfter:
+        - fetch-repository
+      taskRef:
+        kind: ClusterTask
+        name: buildah
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  workspaces:
+    - name: shared-workspace
+----
+
+If you use the Tekton Dashboard on Minikube, Kind or on a Kubernetes cluster, copy the content of the yaml in the pipeline section.
+If you don't want to use the UI, replace the placeholder in the `custom-workflow-image-pipeline.yaml` (the previous yaml) and execute:
+----
+kubectl apply -f custom-workflow-image-pipeline.yaml
+----
+
+== Create a PipelineRun to execute the pipeline
+
+The next step is to create a pipeline run with the details of your git repository and the desired name of the image.
+
+On OCP -> Pipeline menu -> PipelineRun, copy the following yaml file and replace the placeholders with your values.
+
+i.e. `image registry available at image-registry.openshift-image-registry.svc:5000/`
+
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: custom-workflow-image-pipeline-run
+  namespace: custom-namespace
+spec:
+  workspaces:
+    - name: shared-workspace
+      volumeClaimTemplate:
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 500Mi
+  params:
+    - name: deployment-name
+      value: <your-image-name>
+    - name: git-url
+      value: https://<your repo>.git
+    - name: git-revision
+      value: "main"
+    - name: IMAGE
+      value: "image-registry.openshift-image-registry.svc:5000/custom-namespace/<your-image-name>:latest"
+  pipelineRef:
+    name: custom-workflow-image-pipeline
+----
+
+You can check the execution of the pipeline in the UI,
+at the end your image will be available at:
+image-registry.openshift-image-registry.svc:5000/custom-namespace/<your-image-name>:latest
+
+On Minikube, Kind or plain Kubernetes cluster you can install the
+link:{https://tekton.dev/docs/dashboard/}[Tekton Dashboard]
+
+Otherwise replace the in the custom-workflow-image-pipeline-run.yaml and execute
+----
+kubectl apply -f custom-workflow-image-pipeline-run.yaml
+----
+
+
+== Create a PipelineRun with Tekton CLI
+
+If you have installed the Tekton CLI, you can run the pipeline with the following command:
+
+----
```

Comment:
plase take a look in the other adocs and use the same patter for yamls, shell comands, etc..